design/_template.md

```diff
@@ -1,35 +1,37 @@
-# Design proposal template (replace with your proposal's title)
+# Design proposal template `<replace with your proposal's title>`
 
-One to two sentences that describes the goal of this proposal.
-The reader should be able to tell by the title, and the opening paragraph, if this document is relevant to them.
+Status: {Draft,Reviewing,Accepted,Declined}
```

Comment:
per prev comment, I'd prefer not to add this back to our template.

Reply:
done.